apps/api/src/app/integrations/e2e/create-integration.e2e.ts

```diff
@@ -483,12 +483,12 @@ describe('Create Integration - /integration (POST)', function () {
       { sort: { priority: -1 } }
     );
 
-    expect(first._id).to.equal(data._id);
-    expect(first.primary).to.equal(false);
+    expect(first._id).to.equal(activeIntegration._id);
+    expect(first.primary).to.equal(true);
     expect(first.active).to.equal(true);
     expect(first.priority).to.equal(2);
 
-    expect(second._id).to.equal(activeIntegration._id);
+    expect(second._id).to.equal(data._id);
```

Comment:
Reversing the ordering now that the first integration is created as `primary` and will have a lower `priority`.